src/datepicker/datepicker-tools.ts

```diff
@@ -170,9 +174,13 @@ export function buildMonth(
         dayObject = days[day] = {} as DayViewModel;
       }
       dayObject.date = newDate;
-      dayObject.context = Object.assign(
-          dayObject.context || {},
-          {date: newDate, currentMonth: month.number, disabled, focused: false, selected: false});
+      dayObject.context = Object.assign(dayObject.context || {}, {
+        date: newDate,
```

Comment:
Could we take the opportunity here to also introduce `$implicit` usage ?

```ts
$implicit: newDate,
date: newDate, // we can keep that one for backward compat
```

It simply allows people to use the short `let` notation such as:

```html
<ng-template #t let-date>
```

Reply:
Thanks for the suggestion and the review. 
I'll just open another PR with this feature to keep things separate.